common/src/main/java/com/wynntils/features/inventory/ItemScreenshotFeature.java

```diff
@@ -125,10 +125,10 @@ private void takeScreenshot(Screen screen, Slot hoveredSlot, List<Component> ite
 
         if (saveToDisk.get()) {
             // First try to save it to disk
-            String itemNameForFile = WynnUtils.normalizeBadString(ComponentUtils.stripFormatting(
-                            CodedString.fromString(itemStack.getHoverName().getString())))
+            String itemNameForFile = WynnUtils.normalizeBadString(StyledText.fromComponent(itemStack.getHoverName())
```

Comment:
Pretty sure you can do StyledText.fromComponent(..).normalized().getString() instead

Reply:
I didn't know they were doing the same thing